Dockerfile.train.tmpl

```diff
@@ -18,23 +18,25 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         unzip \
         wget
 
+# We need to remove it because it's breaking deepspeech install later with
+# weird errors about setuptools
+RUN apt-get purge -y python3-xdg
+
 WORKDIR /
 RUN git lfs install
 RUN git clone $DEEPSPEECH_REPO
 
 WORKDIR /DeepSpeech
 RUN git checkout $DEEPSPEECH_SHA
 
-# Setup a virtualenv otherwise we mess with the system and this is BAD.
-RUN python3 -m venv venv/
-
-ENV VIRTUAL_ENV=/DeepSpeech/venv
-ENV PATH=$VIRTUAL_ENV/bin:$PATH
-
 # Build CTC decoder first, to avoid clashes on incompatible versions upgrades
 RUN cd native_client/ctcdecode && make NUM_PROCESSES=$(nproc) bindings
 RUN pip3 install --upgrade native_client/ctcdecode/dist/*.whl
 
+# There is already correct TensorFlow GPU installed on the base image,
+# we don't want to break that
+RUN sed -ri "s/'tensorflow == .*',//g" setup.py
+
```

Comment:
This is pretty hacky. Why not add a check to setup.py similar to `DS_NODECODER`?

Reply:
I agree, but because I was worried about if people mixes this Dockerfile with other repo that might not have this kind of change.